tests/ast/data/functions.test

```diff
@@ -712,11 +712,11 @@ df55 = df.select(variance(col("A")))
 
 df56 = df.select(var_pop(col("A")))
 
-df57 = df.select(approx_percentile(col("A"), lit(0.6)), approx_percentile(col("B"), lit(0.0)))
+df57 = df.select()
```

Comment:
These updates don't look OK. I'm reading through the changes in `functions.py` to better understand the pattern. Either AST collection in Python, or the unparser need some fixes.

Reply:
fixed!